birdhouse/config/proxy/conf.d/all-services.include.template

```diff
@@ -7,6 +7,16 @@
         return 200 '${BIRDHOUSE_DEPLOY_COMPONENTS_JSON}';
     }
 
+    location /services {
+        default_type application/json;
+        return 200 '${BIRDHOUSE_DEPLOY_SERVICES_JSON}';
+    }
+
+    location /version {
+        default_type text/plain;
+        return 200 '${BIRDHOUSE_VERSION}';
```

Comment:
Why not return both the version and the release datetime in JSON?

Reply:
Only the version is relevant for our use-case. If you have a use-case that also requires the release time we can add it in a different endpoint.

Reply:
I just though that we might as well provide it since it's already there. The node manager could also validate when was the last update.

Reply:
@fmigneault is having the release datetime a requirement for you? We can adapt the front end to accommodate whatever data you want provided by this endpoint.

Reply:
I find it to be a quicker indication of an "outdated" stack or not, without having to check GitHub when the version was realeased, since we can loose track of the version numbers across all services.